mingw-w64+gcc.sh

```diff
@@ -48,6 +64,8 @@ cd build
 ../src/configure --enable-languages=c,c++ --build=x86_64-w64-mingw32 --host=x86_64-w64-mingw32 \
 --target=x86_64-w64-mingw32 --disable-multilib --prefix=/c/temp/gcc/dest --with-sysroot=/c/temp/gcc/dest \
 --disable-libstdcxx-pch --disable-libstdcxx-verbose --disable-nls --disable-shared --disable-win32-registry \
+--enable-libatomic --enable-threads=posix \
```

Comment:
What's libatomic and why is it related to pthreads? I can't find any documentation about this (after 30 seconds of searching). The last I checked, which was a while ago, GCC's std::atomic seemed to work. Perhaps I'm misremembering.